src/gpg2john.c

```diff
@@ -1629,7 +1697,7 @@ parse_packet(char *hash)
 		if (tag < TAG_NUM && tag_func[tag] != NULL) {
 			if (gpg_dbg)
 				fprintf(stderr, "Packet type %d, len %d at offset %d  (Processing) (pkt-type %s) (Partial %s)\n", tag, len, offset, pkt_type(tag), partial?"yes":"no");
-			(*tag_func[tag])(len, 1, partial, hash);	// first packet (possibly only one if partial is false).
+			(*tag_func1[tag])(len, 1, partial, hash);	// first packet (possibly only one if partial is false).
```

Comment:
We're now calling through a `tag_func1` element, but the NULL check above remained of `tag_func` - that's probably wrong now.